notebook/agentchat_two_users.ipynb

```diff
@@ -176,7 +176,7 @@
     "                    \"properties\": {\n",
     "                        \"message\": {\n",
     "                            \"type\": \"string\",\n",
-    "                            \"description\": \"question to ask expert. Make sure the question include enough context, such as the code and the execution result. The expert does not know the conversation between you and the user, unless you share the conversation with the expert.\",\n",
+    "                            \"description\": \"question to ask expert. Ensure the question includes enough context, such as the code and the execution result. The expert does not know the conversation between you and the user unless you share the conversation with the expert.\",\n",
```

Comment:
While the change makes sense, please be careful when making changes that will affect the results.